python/mp/rosary.py

```diff
@@ -312,7 +312,8 @@ def add_effect(self, *args, **kwargs):
         """
 
         effect_name = kwargs.get('name')
-        bead_set_name = kwargs.get('bead_set', 'all')
+        bead_set_name = kwargs.get('bead_set', 'rosary')
+        bead_set_sort = kwargs.get('bead_set_sort', 'cw')
```

Comment:
I'm encouraging 'rosary' instead of 'all'. I'd like 'all' to actually refer to ALL of the LEDs now that we have expanded scope.